docs/template_plugin/tests/functional/transformations/preprocessing.cpp

```diff
@@ -43,8 +43,8 @@
 //         auto data = std::make_shared<opset5::Parameter>(element::f32, data_shape);
 //         auto scales = opset5::Constant::create(element::f32, scale_shape,
 //             std::vector<float>(shape_size(scale_shape), 2.0f));
-//         auto mul = std::make_shared<opset5::Multiply>(data, scales);
-//         auto relu = std::make_shared<opset5::Relu>(mul);
+//         auto div = std::make_shared<opset5::Divide>(data, scales);
```

Comment:
Why did you change commented code?

Reply:
We aligned scaling for all accessible places. As this code was not deleted, it's potential part of feature.